umake/frameworks/games.py

```diff
@@ -111,42 +111,47 @@ class Unity3D(umake.frameworks.baseinstaller.BaseInstaller):
     def __init__(self, category):
         super().__init__(name="Unity3d", description=_("Unity 3D Editor Linux experimental support"),
                          category=category, only_on_archs=['amd64'],
-                         download_page="https://community.unity.com/t5/" +
-                                       "Linux-Editor/Unity-on-Linux-Release-Notes-and-Known-Issues/m-p/2323665",
+                         download_page="https://forum.unity3d.com/" +
+                                       "threads/unity-on-linux-release-notes-and-known-issues.350256",
                          match_last_link=True,
                          checksum_type=ChecksumType.sha1,
                          dir_to_decompress_in_tarball='unity-editor*',
                          desktop_filename="unity3d-editor.desktop",
                          required_files_path=[os.path.join("Editor", "Unity")],
                          # we need root access for chrome sandbox setUID
-                         need_root_access=True,
+                         need_root_access=False,
```

Comment:
I changed that to run a quick test when I first changed the url...I forgot to change it.
Changed now.